### PR TITLE
Improve code

### PR DESCRIPTION
**Pull request by vinassefranche**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -39,30 +41,38 @@ const parseEnvironmentVariables = async () => {
...
+              .map((repo) => `repo:${repo}`)
+              .join(" ")}`,
+          })
+    ),
+    readerTask.map(({ data: { items } }) =>
+      items.map((pullRequest) => ({
+        title: pullRequest.title,
+        url: pullRequest.html_url,
+        repo: pullRequest.repository_url.replace(
+          "https://api.github.com/repos/inato/",
```
Comment (darya-talanina) [inline src/index.ts]: Minor: This can be extracted in a named variable

File: src/index.ts [hunk 2]
```diff
@@ -103,7 +100,7 @@ const publishToSlack = (
       },
     ]);
   }
-
+  const grouped = groupBy(pullRequests, ({ repo }) => repo);
```
Comment (darya-talanina) [inline src/index.ts]: grouped -> groupedByRepo?

**Review by darya-talanina — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (vinassefranche) [reply src/index.ts]: I even extracted the logic in a function to have it clearer. Thanks

File: src/index.ts (hunk 2 above)
Comment (vinassefranche) [reply src/index.ts]: outdated

**Review by darya-talanina — APPROVED** [review]

**Pull request merged**